ground/src/main/java/com/google/android/ground/ui/offlinebasemap/viewer/OfflineAreaViewerViewModel.kt

```diff
@@ -94,35 +88,7 @@ constructor(
         .toFlowable(BackpressureStrategy.LATEST)
 
     areaName = offlineAreaItemAsFlowable.map(OfflineArea::name).toLiveData()
-    areaStorageSize =
-      offlineAreaItemAsFlowable
-        .flatMap { offlineAreaRepository.getIntersectingDownloadedTileSetsOnceAndStream(it) }
-        .map { mbtilesFiles: Set<MbtilesFile> -> tileSetsToTotalStorageSize(mbtilesFiles) }
-        .toLiveData()
     offlineArea = offlineAreaItemAsFlowable.toLiveData()
-    disposeOnClear(
-      removeAreaClicks
-        .map { offlineArea.getValue()!!.id }
-        .flatMapCompletable { offlineAreaRepository.deleteOfflineArea(it) }
-        .doOnError { Timber.e(it, "Couldn't remove area: %s", offlineAreaId) }
-        .subscribe { navigator.navigateUp() }
-    )
```

Comment:
Since we haven't removed the "Remove" button from the layout, we still need to handle the logic for the MOG tiles.

Reply:
I plan to do that using Coroutines and a simpler impl (tracked as #1788).